public_html/index.html

```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://games.mozilla.org/">
+    <meta property="og:image" content="https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg">
```

Comment:
do you need the secure og:image too? last time I tested, for some reason, I did

Reply:
ohh, i really don't know what the best in this case....

Reply:
I'm actually not entirely sure. IME, docs don't always match implementation.